edx_notifications/server/web/static/edx_notifications/templates/notification_pane.html

```diff
@@ -1,6 +1,35 @@
-<h3>Here are your notifications</h3>
 <ul class='edx-notifications-list'>
     <% _.each(user_notifications, function(user_notification){ %>
         <li class='edx-notifications-list-item'><%= user_notification.html %></li>
     <% }); %>
 </ul>
+<div class="edx-notifications-container">
+    <div class="edx-notifications-content">
+        <h2>Notifications</h2>
+        <div class="actions">
+            <ul class="notifications_list_tab">
+                <li id="unread_notifications" class="active"><a  href="#">View unread</a></li>
```

Comment:
can you use class names rather than id's? You'll have to update the jQuery selectors as well.